org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/desktop/DashboardController.java

```diff
@@ -985,6 +988,17 @@ else if (value instanceof Integer)
 							 iPara.setInfo(info);
 						 }
 					 }
+					 else if (DisplayType.isID(iPara.getDisplayType())) {
```

Comment:
isID is not precise enough since that includes DisplayType.ID, DisplayType.Chart, etc. To handle just table, tabledirr and search, you need to use for just this 3 display type (i.e iPara.getDisplayType() == DisplayType.Table, etc). If the code works for all lookup displaytype, the check should be  DisplayType.isLookup.